src/DesignPatternsLibrary/Program.cs

```diff
@@ -19,6 +19,7 @@ private static void Main(string[] args)
             }
 
             Executors[choiceKey].Execute();
+            Console.ReadLine();
```

Comment:
I like the idea, but I would prefer to notify a user that the action from him is expected in order to continue execution of the program.

What do you think about adding introducing a new method for handling that case? 

```
private static void WaitForContinuationSignal()
    {
        Console.ForegroundColor = ConsoleColor.DarkGreen;
        Console.WriteLine("\n\nExecution results can be found above. Press ENTER to continue.");
        Console.ReadLine();
        Console.ResetColor();
    }
```

Then, we can call a method instead of just doing
`Console.ReadLine();`

Reply:
Ah yes, this is much cleaner!